include/pipert/Scheduler.h

```diff
@@ -29,6 +29,8 @@ class Scheduler {
   void Stop();
   bool IsRunning();
 
+  int GetWorkersNumber();
```

Comment:
This could get a documentation.
Maybe we could call it GetWorkerNumber() instead of "Workers".

Reply:
Replaced name in a new fixup.
Should I add function's documentation to this branch or doc branch?

Reply:
You can add it here as this one will be very simple. And that is it. Thanks a lot!